src/main/java/io/jenkins/security/csp/Scanner.java

```diff
@@ -36,7 +36,9 @@ public class Scanner {
     // Examples indicate trying to match open-paren would be too restrictive:
     // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval#direct_and_indirect_eval
     // geval is defined in hudson-behavior.js
-    protected static final Map<String, Pattern> JS_PATTERNS = Map.of("(g)eval Call", Pattern.compile("\\Wg?eval\\W"));
+    protected static final Map<String, Pattern> JS_PATTERNS = Map.of("(g)eval Call", Pattern.compile("\\Wg?eval\\W"),
+            "Inline Event Handler (JavaScript)", Pattern.compile("[\\s+]" + JS_EVENT_ATTRIBUTES + ".*?((?<!\\\\)['\"]?)[_\\w]+\\s*\\(", Pattern.CASE_INSENSITIVE),
```

Comment:
```suggestion
            "Inline Event Handler (JavaScript)", Pattern.compile("[\\s+]" + JS_EVENT_ATTRIBUTES + "=.*(['"]?)[$_\\w]+\\s*\\(", Pattern.CASE_INSENSITIVE),
```
Could simplify to this. `[$_\\w]+` is not a complete regex for valid JS identifiers though. But maybe good enough?

Can also be
```suggestion
            "Inline Event Handler (JavaScript)", Pattern.compile("[\\s+]" + JS_EVENT_ATTRIBUTES + "=.*(['"]?)", Pattern.CASE_INSENSITIVE),
```
This will match the rest of the line after the handler.